pom.xml

```diff
@@ -191,28 +191,39 @@
 	</build>
 
 	<repositories>
-                <repository>
-                        <id>ulisboa-repo</id>
-                        <url>http://nexjen.ulisboa.pt/nexus/content/groups/public/</url>
-                </repository>
-                <repository>
-                        <id>qubIT-repo-public</id>
-                        <url>https://repository.qub-it.com/nexus/content/groups/public</url>
-                </repository>
-                <repository>
-                        <id>qubIT-repo</id>
-                        <url>https://repository.qub-it.com/nexus/content/groups/qubITRepo</url>
-                </repository>
+		<repository>
+			<id>qubIT-oss</id>
+			<url>https://repository-oss.omnis.cloud/nexus/repository/releases/</url>
+		</repository>
+		<repository>
+				<id>ulisboa-repo</id>
```

Comment:
@joaofreitas-qubit there seems to be some formatting issues here. Can you fix that?